### DISCUSSION TITLE
Toggle to switch mbc and rmc functionality in editor

### DISCUSSION THREAD

**Discussion by ChenMerami** [ppy/osu#29513] [Ideas] [answered]
I think it would be a good idea to add toggle in settings that would allow mappers to switch middle mouse click and right mouse click functionality or maybe just a way to rebind mouse buttons

**Comment by peppy** [comment, marked as answer]
Please get used to it instead.

**Comment by cihe13375** [comment]
FYI, there are programs to customize mouse behavior for specific applications (e.g. X-Mouse Button Control on Windows)
